src/lib/controllers/configuration-manager/index.d.ts

```diff
@@ -0,0 +1,20 @@
+import { ValidationError } from 'jsonschema';
```

Comment:
Does this definition file actually need to exist for external dependencies?

Reply:
The comment is about `ErrorMap` right? We need this type, and it doesn't exist somewhere we can import it from.